extension/content.css

```diff
@@ -468,3 +468,9 @@ hidden content area created to increase hover target
 	text-decoration: none;
 	opacity: 1;
 }
+
+/* collapse multiple discussion items */
+.discussion-item+.discussion-item {
+	padding-top: 0;
+	border-top: none;
```

Comment:
Both of these need more specificity or `!important` since you're using the same selectors GitHub has and they might load in a different order. At least in my test it didn't work until I added `!important`.